addon/-private/adapters/build-url-mixin.js

```diff
@@ -47,16 +47,17 @@ export default Ember.Mixin.create({
     @param {String} modelName
     @param {(String|Array|Object)} id single id or array of ids or query
     @param {(DS.Snapshot|Array)} snapshot single snapshot or array of snapshots
+    @param {(DS.SnapshotRecordArray)} snapshotRecord single snapshotRecordArray
     @param {String} requestType
     @param {Object} query object of query parameters to send for query requests.
     @return {String} url
   */
-  buildURL(modelName, id, snapshot, requestType, query) {
+  buildURL(modelName, id, snapshot, snapshotRecord, requestType, query) {
```

Comment:
You don't need to pass the `snapshotRecord` here, since `snapshot` will either be the single `snapshot` or the `snapshotRecordArray`.

Sorry, I should have made that clear in the other isse :pensive: